libs/judicial-system/types/src/lib/dateLog.ts

```diff
@@ -0,0 +1,26 @@
+export enum DateType {
+  COURT_DATE = 'COURT_DATE',
+}
+
+export const getLatestDateTypeByCaseId = (
+  dateType: DateType,
```

Comment:
I also wonder whether we should take a similar approach to this as we did with the role checks recently where we have quite specific functions, that way the code outside of the dateLog.ts would be cleaner and we wouldnt need to import the DateType everywhere. 
So it would be something like 

getLatestCourtDate(workingCase.dateLogs) or even getCourtDates(workingCase.dateLogs) which returns all the dates in ascending or descending order

Ooor if we made a map or function that filters and sorts date types into more usable entities, like getOrderedDates(workingCase.dateLogs) that would return ordered and grouped date logs that could be used as needed in the components by selecting the first or last of whichever type you need at that point.

Reply:
Actually, another thing we could do is resolve this in the API layer so all youd need to do is use workingCase.courtDate or workingCase.courtDate[0] if its an array, similar as we do with appeal info. That would make the front end so much cleaner